README.md

```diff
@@ -159,7 +159,9 @@ const isStamp = (obj) => {
     typeof obj.compose === 'function';
 };
 
-const stamp = rtype`interface stamp ${ isStamp }`;
+const stamp = rtype(isStamp);
```

Comment:
I'm not quite sure what is going on here. I first thought you simply forgot `(` and `)`. But then why are we passing a string here? Was the plan to tokenize the string inside the `rtype` method? In the future doc you are just passing the actual function to the `rtype` method, so I did the same here.